DEVELOPER_GUIDE.md

```diff
@@ -2,30 +2,46 @@
+- [Executing tests](#executing-tests)
+    - [Unit tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+- [Submitting your changes for a pull request](#submitting-your-changes-for-a-pull-request)
+- [Developing breaking changes](#developing-breaking-changes)
+- [Miscellaneous](#miscellaneous)
+    - [git-secrets](#git-secrets)
+    - [Adding new major and minor Python versions to OpenSearch Benchmark](#adding-new-major-and-minor-python-versions-to-opensearch-benchmark)
+    - [Debugging unittests in Visual Studio Code](#debugging-unittests-in-visual-studio-code)
+
 ## Installation
 
 ### Prerequisites
 
-  - Pyenv : Install `pyenv` and follow the instructions in the output of `pyenv init` to set up your shell and restart it before proceeding.
+  - **Pyenv** : Install `pyenv` and follow the instructions in the output of `pyenv init` to set up your shell and restart it before proceeding.
     For more details please refer to the [PyEnv installation instructions](https://github.com/pyenv/pyenv#installation).
 
-  - JDK: Although OSB is a Python application, it optionally builds and provisions OpenSearch clusters.  JDK version 17 is used to build the current version of OpenSearch.  Please refer to the [build setup requirements](https://github.com/opensearch-project/OpenSearch/blob/ca564fd04f5059cf9e3ce8aba442575afb3d99f1/DEVELOPER_GUIDE.md#install-prerequisites).
+  - **JDK**: Although OSB is a Python application, it optionally builds and provisions OpenSearch clusters.  JDK version 17 is used to build the current version of OpenSearch.  Please refer to the [build setup requirements](https://github.com/opensearch-project/OpenSearch/blob/ca564fd04f5059cf9e3ce8aba442575afb3d99f1/DEVELOPER_GUIDE.md#install-prerequisites).
     Note that the `javadoc` executable should be available in the JDK installation.  An earlier version of the JDK can be used, but not all the integration tests will pass.
 
     ```
     export JAVA_HOME=/path/to/JDK17
 
     ```
 
-  - Install Docker and `docker-compose`.  Start the Docker server.  The user running the integration tests should have the permissions required to run docker commands.  Test by running `docker ps`.
+  - **Docker**: Install Docker and `docker-compose`. Start the Docker server. The user running the integration tests should have the permissions required to run docker commands. Test by running `docker ps`.
 
-  - Git : git 1.9 or later.
+  - **Git** : supports versions 1.9+
 
 ### Setup
 
-Use the following command-line instructions to set up OpenSearch Benchmark for development:
+To develop OSB properly, it is recommended that you fork the official OpenSearch Benchmark repository.
+
+After you git cloned the forked copy of OpenSearch Benchmark, use the following command-line instructions to set up OpenSearch Benchmark for development:
 ```
-git clone https://github.com/opensearch-project/OpenSearch-Benchmark
 cd OpenSearch-Benchmark
```

Comment:
It may be better to use lowercase for consistency, since the GitHub project name is in lowercase.  Else, there can be issues on case-sensitive systems if there is already a directory in the wrong case.

Reply:
Good catch!